freqtrade/freqtradebot.py

```diff
@@ -820,41 +848,41 @@ def check_handle_timedout(self) -> None:
 
             if ((order['side'] == 'buy' and order['status'] == 'canceled')
                     or (self._check_timed_out('buy', order))):
-
                 self.handle_timedout_limit_buy(trade, order)
                 self.wallets.update()
+                order_type = self.strategy.order_types['buy']
+                self._notify_buy_cancel(trade, order_type)
 
             elif ((order['side'] == 'sell' and order['status'] == 'canceled')
                   or (self._check_timed_out('sell', order))):
                 self.handle_timedout_limit_sell(trade, order)
                 self.wallets.update()
+                order_type = self.strategy.order_types['sell']
+                self._notify_sell_cancel(trade, order_type)
 
-    def handle_buy_order_full_cancel(self, trade: Trade, reason: str) -> None:
-        """Close trade in database and send message"""
+    def delete_trade(self, trade: Trade) -> None:
+        """Delete trade in database"""
```

Comment:
not in this PR, please 😆 ???

Reply:
Yes, possibly we can consider doing a PR, but it would probably need to come from you guys, since I'm still new to Python world. I see many other possible refactors needed, we have too many methods that go like check_something_and_do_something_else.

Reply:
yes absolutely - if you look at PR's - we do many refactorings (mostly small PR's)  doing some cleanup, or moving / renaming methods ... so the "not in this PR" message is also pointing to that - this is already a bit ... big :D